include/tdnftypes.h

```diff
@@ -225,6 +226,7 @@ typedef struct _TDNF_CMD_ARGS
     int nNoAutoRemove;     //overide clean_requirements_on_remove config option
     int nJsonOutput;       //output in json format
     int nTestOnly;         //run test transaction only
+    int nSkipBroken;
```

Comment:
Every time adding a new option based on flag we have to use a new variable .Instead if possible can we use a variable by setting the bits.

Reply:
We can refactor that some time. But it's not urgent. It's wasteful because it uses 64 times as much memory (for 64bit integers), but it's still a small amount of memory in total.

Reply:
Sure we can refactor this later on .LGTM